src/components/PropertyLookup.vue

```diff
@@ -64,6 +63,11 @@ export default Vue.extend( {
 				},
 			);
 		},
+		scrollpropertiesSearchResults( newSearchResults: object[] ): void {
```

Comment:
You don't need this watch, you can just move this if-condition as it is into `searchPropertiesOnScroll` and remake `this.scrollpropertiesSearchResults = ...` into a local variable `const searchResults = ...` and it should just work.

Reply:
yes. that was how i implemented it at first. However after it went through the test verification, it wasn't working. you had to scroll up again to see the items loaded again. 
after doing some debugging, i realized that, for some reason, the new scrollpropertiesSearchResults is never updated. i guess because of the async nature of the call?

that is why i ended up adding a watch.

Reply:
good call.
updated